msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -337,6 +337,20 @@ The query parameters are:
 > [!NOTE]
 > You can see `channelId` and `groupId` in the URL from the channel.
 
+### Generate deep links to chat messages
+
+Use this deep link format to navigate to a message in individual and group chats in Teams:
+
+`http://teams.microsoft.com/l/message/{chatId}/{messageId}?context={contextObject}`
+
+Example: `http://dev.local/l/message/19:253f5895-9a62-4362-8d38-43f0205c702c_f1b94dcf-0aa3-4989-bcdf-ef4a5ed00f86@unq.gbl.spaces/1563480968434?context={"contextType"%3A"chat"}`
+
+The query parameters are:
+
+* `chatId`: ChatId that is available from the tab context for group and meeting chat. For example, `19:253f5895-9a62-4362-8d38-43f0205c702c_f1b94dcf-0aa3-4989-bcdf-ef4a5ed00f86@unq.gbl.spaces`.
```

Comment:
I don't actually see where the chat ID is present in a tab. The way I get it is by going to the web version of teams and taking from the url for that chat. The only other option that I am aware of is to use graph to make a call to list chats

Reply:
True, chat ID can be retrieved from the URL of the chat in Teams web client or using the [Graph API ](https://docs.microsoft.com/en-us/graph/api/chat-get?view=graph-rest-1.0&tabs=http). Updated the same in the document.